opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -172,10 +168,9 @@ impl<R: RuntimeChannel> LogProcessor for BatchLogProcessor<R> {
 
         if let Err(err) = result {
             otel_error!(
-                name: "batch_log_processor_emit_error",
-                error = format!("{:?}", err)
+                name: "BatchLogProcessor.Export.Error",
```

Comment:
will this be triggered when channel is full? If yes, we need to rethink this, as this can spam the log output.

Reply:
Yes, this error only triggers when channel is full or closed. We need to add some throttling or logic to prevent flooding - have added the TODO for now, as we need common strategy for such flooding.

Reply:
Agree we need a common strategy, but lets remove the error log from here. It'll flood as-is when buffer is full.

Reply:
should we remove altogether, or make it otel_debug for now - with comment to change it to otel_error once throttling is ready.

Reply:
either of them are fine with me, though I slightly prefer removing altogether, as I don't know if we can ship a throttling solution for next release.